config/blade-icons.php

```diff
@@ -29,6 +29,18 @@
         //     */
         //
         //     'path' => 'resources/svg',
+
+        //     /*
+        //     |-----------------------------------------------------------------
+        //     | Icons disk
+        //     |-----------------------------------------------------------------
+        //     |
+        //     | Provide disk where placed SVG icons. Icons are loaded recursively
+        //     | so there's no need to list every sub-directory.
+        //     |
+        //     */
+        //
+        //     'path' => 'resources/svg',
```

Comment:
This doesn't seems correct?

Reply:
Yes true it will be like

'disk' => 'svg-disk'